client/beefy/src/metrics.rs

```diff
@@ -19,9 +19,9 @@
 //! BEEFY Prometheus metrics definition
 
 use prometheus::{register, Counter, Gauge, PrometheusError, Registry, U64};
-
 /// BEEFY metrics exposed through Prometheus
-pub(crate) struct Metrics {
+#[derive(Clone, Debug)]
+pub struct Metrics {
```

Comment:
also remove `beefy_round_concluded` as it's always the same as `beefy_best_block`